applications/q0/q0_gui.py

```diff
@@ -8,12 +8,18 @@
 from pyqtgraph import PlotWidget, plot
 
 import q0_gui_utils
+from applications.q0.q0_cavity import Q0Cavity
+from applications.q0.q0_cryomodule import Q0Cryomodule
```

Comment:
I don't see the addition of applications.q0.q0_cavity and applications.q0.q0_cryomodule? Only the deletion of q0_linac. 
Am I missing something?

Reply:
Line 19!

Reply:
looks like were missing a q0_cavity.py and q0_cryomodule.py file